src/core/useCubeCamera.tsx

```diff
@@ -0,0 +1,50 @@
+import * as THREE from 'three'
+import { HalfFloatType, Fog, FogExp2, WebGLCubeRenderTarget } from 'three'
+import * as React from 'react'
+import { useMemo } from 'react'
+import { useThree } from '@react-three/fiber'
+
+export type Props = {
+  /** Resolution of the FBO, 256 */
```

Comment:
You'll need to use a more descriptive name here.

Reply:
you mean instead of `resolution` or `Props`?

Reply:
`Props`

Reply:
I've changed it to `CubeCameraOptions`, is it ok?